src/ensembl/src/content/app/entity-viewer/gene-view/components/default-transcripts-list/DefaultTranscriptsList.tsx

```diff
@@ -84,7 +100,10 @@ const DefaultTranscriptslist = (props: Props) => {
         <div className={styles.row}>
           {gene.transcripts.length > 5 && !isFilterOpen && (
             <div className={styles.filterLabel} onClick={toggleFilter}>
-              Filter & sort
+              Filter & sort{' '}
```

Comment:
the empty string shouldn't be necessary. The indicator's horizontal position can be adjusted entirely via CSS.

Reply:
This was done by prettier when formatting the code. I'll remove it nevertheless.

Reply:
✅